birdhouse/config/geoserver/config/proxy/conf.extra-service.d/geoserver.conf.template

```diff
@@ -1,5 +1,16 @@
     location /geoserver/ {
-        proxy_pass http://${PAVICS_FQDN}:8087;
+        proxy_pass http://${PAVICS_FQDN}${TWITCHER_PROTECTED_PATH}/geoserver-web/;
```

Comment:
Would it be easy to have a toogle to disable this?  Meaning a toogle to restore the original proxy config to not go through Twitcher?